components/address-input.jsx

```diff
@@ -577,19 +579,36 @@ var ProvinceSelect = React.createClass({
 
 var Country = React.createClass({
   mixins: [IntlMixin],
-  propTypes: {
-    onChange: React.PropTypes.func.isRequired
-  },
   getInitialState: function() {
     return {
-      valid: true
+      valid: true,
+      country: ""
     };
   },
+  componentDidMount: function() {
+    listener.on("fieldUpdated", this.onFieldUpdated);
+    form.registerField({
+      name: this.props.name,
```

Comment:
You've removed `propTypes` for `onChange` above entirely, but you are still using `this.props.name` here... Can you re-add that back for `name`?